### PR TITLE
doc: fix "the a" typos

### PR DESCRIPTION
**Pull request by jamuir**
I grepped for "the a " and found a few typos in the documentation.

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by nhorman — APPROVED** [review]

**Review by slontis — APPROVED** [review]

File: doc/man3/BIO_f_md.pod [hunk 1]
```diff
@@ -36,7 +36,7 @@ BIO_set_md() sets the message digest of BIO B<b> to B<md>: this
 must be called to initialize a digest BIO before any data is
 passed through it. It is a BIO_ctrl() macro.
 
-BIO_get_md() places the a pointer to the digest BIOs digest method
+BIO_get_md() places a pointer to the digest BIOs digest method
 in B<mdp>, it is a BIO_ctrl() macro.
```
Comment (jamuir) [inline doc/man3/BIO_f_md.pod:40]: "... , it is a BIO_ctrl() macro." should rather  be "... .  It is a BIO_ctrl() macro."

I will fix that.

**Comment by slontis** [comment]
reapproved

**Comment by nhorman** [comment]
reapproved

**Comment by t8m** [comment]
Merged to the master, 3.2, 3.1, and 3.0 branches. Thank you.

**Pull request closed without merging**